ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -95,7 +95,9 @@
 
   @Override
   public void draw(Canvas canvas) {
-    if ((!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0) || mBorderCornerRadii != null) {
+    updatePathEffectForBorderStyle();
+    if ((!CSSConstants.isUndefined(mBorderRadius) && mBorderRadius > 0) || mBorderCornerRadii != null ||
+        !(mBorderStyle == null || mBorderStyle == BorderStyle.SOLID)) {
```

Comment:
I find the condition a bit long and hard to read now. The first line says "there is a border radius set". What does the second line say? Can you extract the new condition into a variable with a clear name for better readability please?